lang/hi.js

```diff
@@ -26,13 +26,13 @@ export default {
 	"components.form-element.defaultError": "{label} अमान्य है।",
 	"components.form-element.defaultFieldLabel": "फ़ील्ड",
 	"components.form-element.input.email.typeMismatch": "ईमेल मान्य नहीं है",
-	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {Number must be greater than {min} and less than {max}.} other {Number must be greater than {min} and less than or equal to {max}.}}} other {{maxExclusive, select, true {Number must be greater than or equal to {min} and less than {max}.} other {Number must be greater than or equal to {min} and less than or equal to {max}.}}}}",
-	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {संख्या को इससे कम {max} होना आवश्यक है।} other {संख्या को इससे कम या इसके बराबर {max} होना आवश्यक है।}}",
-	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {संख्या को इससे बड़ा {min} होना आवश्यक है।} other {संख्या को इससे बड़ा है या इसके बराबर {min} होना आवश्यक है।}}",
+	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {संख्या {min} से ज़्यादा और {max} से कम होनी चाहिए.} other {संख्या {min} से ज़्यादा और {max} से कम या उसके बराबर होनी चाहिए.}}} other {{maxExclusive, select, true {संख्या {min} से ज़्यादा या उसके बराबर और {max} से कम होनी चाहिए.} other {संख्या  {min} से ज़्यादा या उसके बराबर और {max} से कम या उसके बराबर होनी चाहिए.}}}}",
+	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {संख्या {max} से कम होनी चाहिए.} other {संख्या {max} से कम या उसके बराबर होनी चाहिए.}}",
```

Comment:
Before `The number needs to be less than {max}.` After `The number must be less than {max}.` After seems correct

Before `The number needs to be less than or equal to {max}.` After `The number must be less than or equal to {max}.`. After seems correct